src/ciloptions.ml

```diff
@@ -167,6 +167,15 @@ let options : (string * Arg.spec * string) list =
                      " Same as --check, but treats problems as errors not warnings.";
     "", Arg.Unit (fun _ -> ()), "";
 
+    "--cstd", Arg.String (fun s -> Cilutil.cstd := Cilutil.cstd_of_string s),
+                      " Specify the c language standard. Choose between c90, c99, c11 and gnu89inline";
+
+    "--mergeinlines", Arg.Unit (fun _ -> Mergecil.merge_inlines := true),
+                      " Try to merge definitions of inline functions. They can appear in multiple
+                      files and we would like them all to be the same. This can slow down the
+                      merger an order of magnitude.";
+    "", Arg.Unit (fun _ -> ()), "";
```

Comment:
What purpose does this empty argument have?

Reply:
I checked that they do not make any difference in the output of `--help`, so I removed them. (There are a few similar ones like `    "", Arg.Unit (fun () -> ()), " \n\t\tGeneral Options\n";` that actually do make a difference, because they produce headers for groups of options.)